core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/ssp_config.h

```diff
@@ -42,28 +42,30 @@
 #define CONFIG_HAS_GETRANDOM 0
 #endif
 
-#if defined(__CloudABI__) || defined(BH_PLATFORM_FREERTOS)
+#if defined(__CloudABI__) || defined(BH_PLATFORM_FREERTOS) \
+    || defined(BH_PLATFORM_ZEPHYR)
 #define CONFIG_HAS_CAP_ENTER 1
 #else
 #define CONFIG_HAS_CAP_ENTER 0
 #endif
 
 #if !defined(__APPLE__) && !defined(__FreeBSD__) && !defined(__EMSCRIPTEN__) \
     && !defined(ESP_PLATFORM) && !defined(DISABLE_CLOCK_NANOSLEEP)           \
-    && !defined(BH_PLATFORM_FREERTOS)
+    && !defined(BH_PLATFORM_FREERTOS) && !defined(BH_PLATFORM_ZEPHYR)
 #define CONFIG_HAS_CLOCK_NANOSLEEP 1
 #else
 #define CONFIG_HAS_CLOCK_NANOSLEEP 0
 #endif
 
-#if defined(__APPLE__) || defined(__CloudABI__)
+#if defined(__APPLE__) || defined(__CloudABI__) && !defined(BH_PLATFORM_ZEPHYR)
```

Comment:
`defined(__APPLE__) || defined(__CloudABI__)` means `BH_PLATFORM_ZEPHYR` isn't defined, should not change this line? Or do you mean `|| defined(BH_PLATFORM_ZEPHYR)`?

Reply:
This was for readability, and you're right. I will go back to the original version.